modules/migration-understanding-data-copy-methods.adoc

```diff
@@ -28,6 +28,12 @@ a|* Slower than the snapshot copy method.
 * Optional data verification significantly reduces performance.
 |===
 
+====
+[NOTE]
+Our Restic and Rynsc PV migration assumes that the PVs supported are only volumeMode=filesystem. We *do not* support volumeMode=Block for filesystem migration.
```

Comment:
```suggestion
Our Restic and Rsync PV migration assumes that the PVs supported are only volumeMode=filesystem. We *do not* support volumeMode=Block for filesystem migration.
```

Reply:
I corrected the typo. Thanks for noticing it.